bom/runtime/pom.xml

```diff
@@ -976,6 +976,12 @@
                 <groupId>com.github.ben-manes.caffeine</groupId>
                 <artifactId>caffeine</artifactId>
                 <version>${caffeine.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>org.checkerframework</groupId>
+                        <artifactId>checker-qual</artifactId>
+                    </exclusion>
+                </exclusions>
```

Comment:
If we know it's an issue, can we add it to the banned dependencies of the enforcer in the build-parent?

Reply:
It also goes in through maven-embedder but then it is only used in deployment. 
I'd need to exclude it there too to ban it.
I'll give it a try.